### ISSUE TITLE
Syntax error zeroconf.py

### ISSUE THREAD

**Issue by beidem** [masmu/pulseaudio-dlna#364]
Installed in virtualenv. When i try to run bin/pulseaudio-dlna i get this:
`File "/home/beidem/Nedlastingar/pulseaudio-dlna/lib/python2.7/site-packages/zeroconf.py", line 17`
`      def current_time_millis() -> float:`
`                                             ^`
`SyntaxError: invalid syntax`

**Comment by alazyworkaholic** [comment]
I had the same error. I had simply installed via apt install pulseaudio-dlna in Ubuntu 18.04.
For some reason it was looking up the zeroconf at /usr/local/lib/python2.7/dist-packages/zeroconf.py
I don't even know why /usr/local/lib/python{2.7,3.6} existed on my system since I think everything should only be within /usr/lib/python{2.7,3.6}, but I simply rm'd it and it worked fine.
For whatever reason, the zeroconf in /usr/local/lib/python2.7 was actually a newer version 0.21.3 that uses the type hinting features ( -> float ) that caused the error. My standard ubuntu install worked fine.

**Comment by jfriar** [comment]
From the zeroconf pypi page ( https://pypi.org/project/zeroconf/ ):

> 0.20.0
> Dropped support for Python 2 (this includes PyPy) and 3.3

setup.py in the pulseaudio-dlna package can be adjusted - I changed:

> "zeroconf >= 0.17.4",

to

> "zeroconf <= 0.19.1",

and then ran make.

**Comment by jfriar** [comment]
Also noted in #342

**Comment by blueyed** [comment]
Yes, duplicate of https://github.com/masmu/pulseaudio-dlna/issues/342.
